package.json

```diff
@@ -6,7 +6,6 @@
   "version": "1.7.0-beta.4",
   "main": "dist/lib/auth0-spa-js.cjs.js",
   "types": "dist/typings/index.d.ts",
-  "browser": "dist/auth0-spa-js.production.js",
```

Comment:
Why is this removed?

Reply:
Webpack resolves the library by checking the package properties in this order: "browser" -> "module" -> "main"

This is designed so that if you have a library that provides a different flavour for nodejs and the browser (like `axios`)

We only have one flavour, browser, but we do want to provide an optimized es module version to webpack (or any build tool that prefers es modules). So if we delete "browser" webpack will use module and get the benefits of esm (tree shaking, named exports), things that use commonjs (like browserify) will use "main"